chef/cookbooks/tempest/recipes/config.rb

```diff
@@ -230,21 +231,22 @@
 alt_flavor_ref = "7"
 heat_flavor_ref = "8"
 
+novacmd = "nova --insecure --os_username #{tempest_adm_user} --os_password #{tempest_adm_pass} --os_tenant_name #{tempest_comp_tenant} --os_auth_url #{keystone_settings["internal_auth_url"]}"
```

Comment:
ah, I see. hmm, moving the nova (node) definition until after this spot would solve the issue, but is maybe also not nice. :/